Dockerfile

```
@@ -90,9 +91,19 @@ RUN cd /app/parsec && \
 
+# Install Kani.
 ENV CARGO_HOME="/app/.cargo"
 ENV RUSTUP_HOME="/app/.rustup"
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 ENV PATH="/app/.cargo/bin:${PATH}"
 RUN rustup default 1.90.0
 RUN cargo install --locked kani-verifier && cargo kani setup
+
+# Install ParseRust
+RUN rustup install nightly-2024-10-01
+RUN rustup default nightly-2024-10-01
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)